editors/vscode/package.json

```diff
@@ -163,6 +163,11 @@
             }
           },
           "description": "Map of paths in which the `import` statement for `@mylibrary` imports are looked up. This is an object such as `{\"mylibrary\": \"/path/to/library\"}`. Relative paths are resolved against the workspace root."
+        },
+        "slint.searchForLspInPath": {
```

Comment:
Can we also have an option to set the full path to the LSP ?

Reply:
Same here, I think an option to specify the full path **instead** of a boolean is better, because it's 100% deterministic (unlike PATH) and it makes more sense in the context of VS code's command line launch semantics. (running `code /path` isn't guaranteed to pick up `PATH`).

Reply:
The original reporter was using nixos: They will have the slint-lsp binary stored in the nix store. A part of the file path will be a hash value that will change on each update. Nix-shell (or whatever that thing is called) will typically set up PATH to contain a *long* list of entries to bring everything the user had configured into scope.

For that use-case I think the boolean I implemented makes more sense.

We can add another setting with a full file path. What should be the semantics when that is set? *Only* try that file and error out when it is not found? Or append it as the first entry in the list of file paths to check?

What should we do with the boolean when we add the full file path? Remove that again?

Reply:
Not all the nixos users will have slint-lsp in their path or using devshells. 

That's the beauty of Nixos which is amazing when it comes to containerization.

For example, if we set the configuration option for the slint-lsp path to something like `${lib.getExe pkgs.slint-lsp}`, the LSP will only be available there (in vscode) and not in the user path. Which makes sense, it's very rare when users have to deal themselves with a LSP, it's usually their IDE that does the job.

Reply:
The newest version has a string setting that is either empty (what we did before), an absolute path (use exactly that binary) or a relative path (look for this in $PATH).

Reply:
Excellent, thanks !